docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -945,9 +945,9 @@ Hard limit is configured via system tools
 
 ## database_atomic_delay_before_drop_table_sec {#database_atomic_delay_before_drop_table_sec}
 
-Sets the delay before remove table data in seconds. If the query has `SYNC` modifier, this setting is ignored.
+The delay before a table data is dropped in seconds. If the `DROP TABLE` query has a `SYNC` modifier, this setting is ignored.
```

Comment:
Someone who doesn't know the UNDROP statement may be confused if DROP always incurs a (synchronous) delay of 8 minutes. Maybe rephrase like this?

```
The delay during which a dropped table can be restored using the UNDROP statement. If `DROP TABLE` ran with a `SYNC` modifier, the setting is ignored.
```

(ideally, you link UNDROP with https://clickhouse.com/docs/en/sql-reference/statements/undrop)